assets/helpers/page/page.js

```diff
@@ -49,6 +57,16 @@ function analyticsInitialisation(participations: Participations): void {
   ga.setDimension('experience', abTest.getVariantsAsString(participations));
   ga.trackPageview();
 
+  const dimensions:Dimensions = {
```

Comment:
There's no point creating this const if DNT is true, I'd move it inside the if block.